datetime_tz/__init__.py

```diff
@@ -123,12 +150,92 @@ def localtz_set(timezone):
   global _localtz
   _localtz = _tzinfome(timezone)
 
+def require_timezone(zone):
+  """Raises an AssertionError if we are not in the correct timezone"""
+  assert localtz().zone == zone,\
+      "Please set your local timezone to %(zone)s (either in the machine,\
+      or on Linux by exporting TZ=%(zone)s" % {"zone": zone}
+
+# The following code is a workaround to
+# GetDynamicTimeZoneInformation not being present in win32timezone
+
+class SYSTEMTIME_c(ctypes.Structure):
```

Comment:
I can move require_timezone out into one of our own libraries, as we originally added it as a test case sanity assert, for those tests which need to be run in certain timezones to check specific issues.  Just to verify, you;d like require_timezone to be removed, and the ctypes Structures and _detect_timezone_windows to be moved to another file?